replications/internal/queue_management.go

```diff
@@ -23,14 +23,20 @@ type durableQueueManager struct {
 func NewDurableQueueManager(log *zap.Logger, enginePath string) *durableQueueManager {
 	replicationQueues := make(map[platform.ID]*durablequeue.Queue)
 
+	queuePath := filepath.Join(
+		enginePath,
+		"replicationq",
```

Comment:
I notice in the tests that this `replicationq` string is used a lot. We could avoid it if we pushed this `filepath.Join` out of `NewDurableQueueManager` up one level to the call-site in `replications/service.go`, and made `NewDurableQueueManager` take a `queuePath` instead of an `enginePath`. I think it still makes sense for the `os.MkdirAll` to live in this function.